extensions/amp-bind/0.1/bind-expr-impl.jison

```diff
@@ -279,7 +279,9 @@ literal:
 primitive:
     STRING
       %{
-        const string = yytext.substr(1, yyleng - 2);
+        // Replace leading/trailing single-quote with double-quote chars and
+        // use JSON.parse() to process special chars e.g. '\n'.
+        const string = parseJson(`"${yytext.substr(1, yyleng - 2)}"`);
```

Comment:
Is this safe? What if we used a single quote and the string includes double quotes inside it?

Reply:
Attribute values can't have double quotes.

Reply:
Note, this will also transform unicode escapes (`\u0000`) into their unicode chars.

Reply:
Yea, that's #22265.

Reply:
`<div attr='foo"bar'>` is legal HTML

Reply:
Excellent catch. 👍